Examples/Algorithms/TruthTracking/ActsExamples/TruthTracking/ParticleSmearing.hpp

```diff
@@ -61,10 +61,20 @@ class ParticleSmearing final : public IAlgorithm {
     double sigmaPRel = 0.05;
     /// Optional. Initial covariance matrix diagonal. Overwrites the default if
     /// set.
-    std::optional<std::array<double, 6>> initialSigmas = std::array<double, 6>{
-        1 * Acts::UnitConstants::mm,     1 * Acts::UnitConstants::mm,
-        1 * Acts::UnitConstants::degree, 1 * Acts::UnitConstants::degree,
-        0.1 / Acts::UnitConstants::GeV,  1 * Acts::UnitConstants::ns};
+    std::optional<std::array<double, 6>> initialSigmas;
+    /// Initial q/p coefficient covariance matrix diagonal.
+    std::array<double, 6> initialSimgaQoverPCoefficients = {
+        0 * Acts::UnitConstants::mm /
+            (Acts::UnitConstants::e * Acts::UnitConstants::GeV),
+        0 * Acts::UnitConstants::mm /
+            (Acts::UnitConstants::e * Acts::UnitConstants::GeV),
+        0 * Acts::UnitConstants::degree /
+            (Acts::UnitConstants::e * Acts::UnitConstants::GeV),
+        0 * Acts::UnitConstants::degree /
+            (Acts::UnitConstants::e * Acts::UnitConstants::GeV),
+        0,
+        0 * Acts::UnitConstants::ns /
+            (Acts::UnitConstants::e * Acts::UnitConstants::GeV)};
```

Comment:
They're all just always zero?

Reply:
I changed this significantly in https://github.com/acts-project/acts/pull/3422 where I pulled changes from here. Will factor this out here